### DISCUSSION TITLE
Text encode steps

### DISCUSSION THREAD

**Discussion by mistersprinklez** [TheLastBen/fast-stable-diffusion#1274] [General]
Question:

How many text encode steps should i use with a dataset of 2000 images and about 200 different filenames?

**Comment by TheLastBen** [comment]
start with 800 steps textenc at 1e-6

**Reply by mistersprinklez** [reply]
What would you say for a dataset with 2000 images and only 3 different filenames?

**Reply by TheLastBen** [reply]
then 300-400 steps would suffice, you can go lower, but to be safe, keep it above 300
